pdfquery/pdfquery.py

```diff
@@ -25,36 +25,51 @@
 from pdftranslator import PDFQueryTranslator
 from cache import DummyCache
 
+
 # Re-sort the PDFMiner Layout tree so elements that fit inside other elements
 # will be children of them
-
 def _append_sorted(root, el, comparator):
-    """ Add el as a child of root, or as a child of one of root's children. Comparator is a function(a, b) returning > 0 if a is a child of b, < 0 if b is a child of a, 0 if neither. """
+    """ Add el as a child of root, or as a child of one of root's children.
+    Comparator is a function(a, b) returning > 0 if a is a child of b, < 0 if
+    b is a child of a, 0 if neither.
+    """
     for child in root:
         rel = comparator(el, child)
-        if rel > 0: # el fits inside child, add to child and return
+        if rel > 0:
+            # el fits inside child, add to child and return
             _append_sorted(child, el, comparator)
             return
-        if rel < 0:  # child fits inside el, move child into el (may move more than one)
+        if rel < 0:
+            # child fits inside el, move child into el (may move more than one)
             _append_sorted(el, child, comparator)
     # we weren't added to a child, so add to root
     root.append(el)
 
+
 def _box_in_box(el, child):
     """ Return True if child is contained within el. """
-    return float(el.get('x0')) <= float(child.get('x0')) and float(el.get('x1')) >= float(child.get('x1')) and float(el.get('y0')) <= float(child.get('y0')) and float(el.get('y1')) >= float(child.get('y1'))
+    return all([
+        float(el.get('x0')) <= float(child.get('x0')),
+        float(el.get('x1')) >= float(child.get('x1')),
+        float(el.get('y0')) <= float(child.get('y0')),
+        float(el.get('y1')) >= float(child.get('y1')),
+    ])
 
-_comp_bbox_keys_required = set(['x0', 'x1', 'y0', 'y1'])
 
+_comp_bbox_keys_required = {'x0', 'x1', 'y0', 'y1'}

```

Comment:
This is done the old-fashioned way to maintain Python 2.6 compatibility (bug #16).